src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -663,7 +654,6 @@ private void remove(Resident resident) {
 						if (newMayor != resident) {
 							try {
 								setMayor(newMayor);
```

Comment:
I don't know why this continue was removed. Would this not end up making each resident mayor until the last one is finally made mayor?